teacher.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'person'
+class Teacher < Person
+  def initialize(name, age, parent_permission, specialization)
```

Comment:
- Optional
I would be great if you would provide a default value of `unknown` for name and `true` for parent_premission, so, even if a name or parent_premission is not provided, it will not raise an error since you have set default value for that case.